client/verta/docs/api/api/registry.rst

```diff
@@ -0,0 +1,18 @@
+Model Registry
+==============
+
+Client
+------
+.. automethod:: verta.Client.get_or_create_registered_model
+.. automethod:: verta.Client.get_registered_model
+.. automethod:: verta.Client.get_registered_model_version
```

Comment:
Sphinx's autodoc extension will fetch the parameter and docstring for this method